widget/embedded/src/components/SwapDetails/SwapDetails.helpers.tsx

```diff
@@ -52,7 +52,36 @@ export const getSteps = ({ swap, blockchains, ...args }: GetStep): Step[] => {
           ),
         },
       },
-      swapper: { displayName: step.swapperId, image: step.swapperLogo ?? '' },
+      swapper: {
+        displayName: step.swapperId,
+        image: step.swapperLogo ?? '',
+        type: step.swapperType,
+      },
+      internalSwaps: step.internalSwaps
+        ? step.internalSwaps.map((internalSwap) => {
+            return {
+              from: {
+                blockchain:
+                  getBlockchainShortNameFor(
+                    internalSwap.fromBlockchain,
+                    blockchains
+                  ) ?? '',
+              },
+              to: {
+                blockchain:
+                  getBlockchainShortNameFor(
+                    internalSwap.toBlockchain,
+                    blockchains
+                  ) ?? '',
+              },
+              swapper: {
+                displayName: internalSwap.swapperId,
```

Comment:
We have a function called `getSwapperDisplayName`. Would it be possible to get the title instead of the id from that function? Also in main swapper.